src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+<Page
```

Comment:
Should this be a UserControl?

Reply:
Could be but doesn't have to be, it's never embedded into another page just used for the flyout.